README.md

```diff
@@ -42,22 +42,22 @@ go install github.com/caddyserver/xcaddy/cmd/xcaddy@latest
 Example Caddyfile (replace `user` and `pass` accordingly):
 ```
 {
-  servers {
-    protocols h1 h2 h3
-  }
+	order forward_proxy before file_server
+	servers :443 {
+		protocols h1 h2 h3
+	}
 }
-:443, example.com
-tls me@example.com
-route {
-  forward_proxy {
-    basic_auth user pass
-    hide_ip
-    hide_via
-    probe_resistance
-  }
-  file_server {
-    root /var/www/html
-  }
+:443, example.com {
+	tls 123456@mail.com #Optional
```

Comment:
`me@example.com`

`example.com` is required to avoid copy-paste errors.

Not needed to indicate optional here. The purpose of the example is to provide an optimal default, not to explain options.